pom.xml

```diff
@@ -1952,7 +1952,9 @@ flexible messaging model and an intuitive client API.</description>
                   <goal>report</goal>
                 </goals>
                 <configuration>
-                  <outputDirectory>target/report</outputDirectory>
+                  <excludes>
+                    <exclude>META-INF/**</exclude>
+                  </excludes>
```

Comment:
I don't know the reason you made this change. Could you explain it a bit @yaalsn?

[The last time](https://github.com/apache/pulsar/pull/17382#discussion_r971454409) you changed to `**/*.jar` and now a new change, we need some background here.

However, given that the CI result looks good, I assume this change is OK.

Reply:
@tisonkun You can take a look at [this one](https://stackoverflow.com/questions/50163426/jacoco-cannot-exclude-classes). Third-party classes and jars are in target/META-INF path, so we need to exlude this path.

Reply:
@yaalsn Thank you. One more question: why remove `<outputDirectory>target/report</outputDirectory>`?

Reply:
@tisonkun Because I think the [default value](https://www.eclemma.org/jacoco/trunk/doc/report-mojo.html) makes sense: `target/jacoco` directory means jacoco reports but `target/report` means multi kinds of reports.

Reply:
SGTM :)

Reply:
@tisonkun Thanks for your comments! I think all the questions are very good which I met during this PR, others see the discussion can know clearly about these.